rig.js

```diff
@@ -680,11 +680,11 @@ class RigManager {
 
     this.peerRigs = new Map();
     
-    this.lastTimestamp = Date.now();
+    this.lastTimestamp = performance.now()
```

Comment:
This is probably a good change, but are you sure nothing downstream is expecting integers?

Reply:
rework in the process, but this change only adds accuracy to the time, I'm sure there shouldn't be any problems below